### ISSUE TITLE
Nightwatch not executing in pipeline

### ISSUE THREAD

**Issue by jaswinsingh** [acquia/blt#4630]
I'm trying to execute the Nightwatch test on Acquia pipeline for every PR raised

I have created a custom blt command that execute the test cases of nightwatch. it is executing fine locally but it fails on the pipeline. 

**Detailed error output**

[Tests/Toolbar Test] Test Suite

- Connecting to localhost on port 9515...
 Error connecting to localhost on port 9515.

[Tests/Toolbar Test] Test Suite

- Connecting to localhost on port 9515...
 Error connecting to localhost on port 9515.

Error: An error occurred while retrieving a new session: "session not created: This version of ChromeDriver only supports Chrome version 98"
       at HttpRequest.emit (node:events:513:28)
       at HttpResponse.emit (node:events:513:28)
       at IncomingMessage.emit (node:events:525:35)
       at endReadableNT (node:internal/streams/readable:1358:12)
       at processTicksAndRejections (node:internal/process/task_queues:83:21)
  Error: An error occurred while retrieving a new session: "session not created: This version of ChromeDriver only supports Chrome version 98"
       at HttpRequest.emit (node:events:513:28)
       at HttpResponse.emit (node:events:513:28)
       at IncomingMessage.emit (node:events:525:35)
       at endReadableNT (node:internal/streams/readable:1358:12)

**System information**
* BLT version: 12.8.2.0

**Comment by mikemadison13** [comment]
A couple of things, could you please update to BLT 13.x? 

Also, could you confirm if you have a ci.blt.yml file in your blt directory?

**Comment by jaswinsingh** [comment]
@mikemadison13 Thanks for the quick response, Yes there is `ci.blt.yml` in the blt directory

**Comment by mikemadison13** [comment]
other than upgrading to blt 13.x, i'm curious if you have https://github.com/acquia/blt-drupal-test in the codebase? it does have nightwatch specific functionality.

**Comment by jaswinsingh** [comment]
@mikemadison13 Yes, we were using https://github.com/acquia/blt-drupal-test package to execute the test cases, but it was failing for that too, Also we are upgrading to BLT 13.x shortly.
